dev/prepare_env.bat

```diff
@@ -1,11 +1,9 @@
 rem Last tested with Python 3.8
 
 rem Anaconda needs to be installed upfront
-conda update -n base -c defaults conda
-conda create -y --name tvb-widgets python=3.8 nomkl numba scipy numpy jupyterlab
-conda install -y --name tvb-widgets -c conda-forge pyvista
-
-pip install --upgrade pip
-pip install -r ../requirements.txt
+conda update -n base -c defaults conda & ^

```

Comment:
This ending will continue if a previous command end in error. I don't think this is what we want.
Rather we should stop in case of an error